src/test/kotlin/com/fasterxml/jackson/module/kotlin/test/github/failing/Github50.kt

```diff
@@ -19,10 +19,8 @@ class TestGithub50 {
     @Test
```

Comment:
Should this test be moved from under "failing" to main package?

Reply:
Word. Good catch thanks!